ManatiCR/FDI
# FDI-24 machotes

[FDI-24](https://manati.atlassian.net/browse/FDI-24)
[FDI-25](https://manati.atlassian.net/browse/FDI-25)
-
**Steps to test**
- Run `ahoy drush cim -y && ahoy drush cr`
- Log in as an admministrador de contenido user
- Create one node machote
- Go to http://fdi.docker/es/admin/fdi/machotes
- Check you can edit it and delete it as well and that the page looks fine
- Check the FDI admin page (http://fdi.docker/en/fdi/admin) looks as expected and that the FDI links go to the right links
- Ensure you can access to the required directions only as administrador de contenido user and super admin
